static/css/ubuntu.css

```
@@ -1,27 +1 @@
-@font-face {
-    font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Regular.woff2') format('woff2');
-    font-display: swap;
-    font-weight: 100 400;
-}
-
-@font-face {
-    font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Bold.woff2') format('woff2');
-    font-display: swap;
-    font-weight: 800;
-}
-
-@font-face {
-    font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Light.woff2') format('woff2');
-    font-display: swap;
-    font-weight: 200;
-}
-
-@font-face {
-    font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Medium.woff2') format('woff2');
-    font-display: swap;
-    font-weight: 500;
-}
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@400&display=swap');
```

Ubunto 700 not needed?
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@400&display=swap');
```

---

Ubuntu 700 is coming from the quill-tailwind font css @habib-deriv